semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Main.scala

```diff
@@ -28,6 +28,14 @@ object Main {
     val semanticdbRoot = AbsolutePath(settings.d).resolve("META-INF").resolve("semanticdb")
     Files.createDirectories(semanticdbRoot.toNIO)
     var failed = false
+    def fail(file: Path, ex: Throwable): FileVisitResult = {
+      println(s"error: can't convert $file")
+      if (ex != null) {
+        ex.printStackTrace()
+      }
```

Comment:
How can ex be null here?

Reply:
In postVisitDirectory, 

```
     * @param   exc
     *          {@code null} if the iteration of the directory completes without
     *          an error; otherwise the I/O exception that caused the iteration
     *          of the directory to complete prematurely
```

Reply:
If `ex` being `null` indicates success, then you can't call `fail` from `postVisitDirectory`.

Reply:
guess failed should not become false in that case, let me fix this asap

Reply:
Comforting to know CI caught this bug https://ci.appveyor.com/project/olafurpg/scalameta/build/744#L1797